### PR TITLE
metamask mobile fix

### PR DESCRIPTION
**Pull request by dankins**
* updates web3 to 1.2.4
* bumps kirby version to bring in a change made in the view plugin need for :point_down: 
* removes reach router - for some reason whenever a kirby child-app view called "navigate" it would completely bork web3. this refactors the child app to just use the view plugin without a routing layer

### CODE REVIEW

File: packages/kirby/src/viewport/Viewport.tsx [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
-    <Router>
-      <Web3Enable path="/ethereum/web3enable/:network" />
-      <SignatureConfirm path="/ethereum/confirm-signature" />
-      <Login path="/identity/login" />
-      <Signup path="/identity/signup" />
-    </Router>
-  );
+  const view = useKirbySelector((state: any) => state.view.queue);
+
+  console.log("view is:", view);
```
Comment (nickreynolds) [inline packages/kirby/src/viewport/Viewport.tsx]: want to leave this in for now?

Comment (dankins) [reply packages/kirby/src/viewport/Viewport.tsx]: ah no, I'll remove

**Comment by dankins** [comment]
:tada: This PR is included in version 4.9.3 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**